napari/components/_tests/test_layer_slicer.py

```diff
@@ -83,6 +83,7 @@ class LockableData:
     def __init__(self, data: LayerDataProtocol):
         self.data = data
         self.lock = RLock()
+        self.ndim = data.ndim
```

Comment:
Required to avoid `ValueError: Input data should be an array-like object, or a sequence of arrays of decreasing size. Got arrays of single shape: (10, 15)` inside of `layer.guess_multiscale()`

Reply:
Normally, passing `multiscale=False` in the layer initializer should be enough to avoid that error. But I'm not against adding this if needed.

Reply:
If this is needed, then maybe just make this a derived property that comes from `self.data.ndim`?

Reply:
because of the nonlinear `if` checking, even setting `multiscale=False` doesn't avoid this message. I can change it to a property derived from `self.data.ndim` though.